.idea/modules.xml

```diff
@@ -2,8 +2,9 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
+      <module fileurl="file://$PROJECT_DIR$/Envelop.iml" filepath="$PROJECT_DIR$/Envelop.iml" />
```

Comment:
The changes on this file are probably not required, can you revert or ignore them?